src/enftun.c

```diff
@@ -253,7 +269,9 @@ enftun_connect(struct enftun_context* ctx)
     enftun_tls_disconnect(&ctx->tls);
 
 close_conn_state:
+#ifndef USE_PSOCK
```

Comment:
Uggh, so many `#ifdef USE_PSOCK`/`#ifndef USE_PSOCK` lines just for the conn_state stuff.    These will get removed anyway, since we want to manage the connection state is both cases. But if we didn't, a better way of encapsulating these differences would be needed.